app/parse.py

```diff
@@ -16,7 +31,113 @@ class Product:
 
 
 def get_all_products() -> None:
-    pass
+    pages = [
+        {"url": "", "name": "home"},
+        {"url": "computers", "name": "computers"},
+        {"url": "phones", "name": "phones"},
+        {"url": "computers/laptops", "name": "laptops",
+         "additional_url": "test-sites/e-commerce/more/computers/laptops"},
+        {"url": "computers/tablets", "name": "tablets",
+         "additional_url": "test-sites/e-commerce/more/computers/tablets"},
+        {"url": "phones/touch", "name": "touch",
+         "additional_url": "test-sites/e-commerce/more/computers/touch"}
+    ]
+    options = Options()
+    service = Service("path/to/chromedriver")
+    driver = webdriver.Chrome(service=service, options=options)
+
+    driver.get(HOME_URL)
+    time.sleep(2)
+
+    accept_cookies_button = driver.find_elements(
+        By.CSS_SELECTOR, ".acceptCookies"
+    )
+    if accept_cookies_button:
+        accept_cookies_button[0].click()
+        time.sleep(2)
+
+    for page in pages:
+        url = urljoin(HOME_URL, page["url"])
+
+        if "additional_url" in page:
+            additional_url = urljoin(BASE_URL, page["additional_url"])
+            scrape_page(driver, [], page["name"], additional_url)
+        else:
+            scrape_page(driver, [], page["name"])
```

Comment:
No need to use if statement. You can use .get()
```suggestion
        additional_url = page.get("additional_url")
        dynamic_url = additional_url and urljoin(BASE_URL, additional_url)
        scrape_page(driver, [], page["name"], dynamic_url)
```
Also why you pass empty list to scrape_page? Maybe just return list from scrape_page?